scripts/kconfig/kconfigfunctions.py

```diff
@@ -388,6 +388,32 @@ def dt_node_int_prop(kconf, name, path, prop, unit=None):
     if name == "dt_node_int_prop_hex":
         return hex(_node_int_prop(node, prop, unit))
 
+def dt_node_prop_equal_str(kconf, _, path, prop, val):
```

Comment:
I think for consistency with dt_node_int_prop_foo, this should be called dt_node_str_prop_equals or something like that.

Reply:
Changed to dt_node_str_prop_equals.